Makefile

```diff
@@ -89,6 +93,16 @@ revive-alerting: scripts/go/bin/revive
 		-formatter stylish \
 		./pkg/services/alerting/...
 
+bra: scripts/go/bin/bra
```

Comment:
I wonder if we can come up with a better name :)
With this change I dont think we have to care about how the process start. Perhaps we could name it something more in the lines of "start grafana and watch for changes" :)

Reply:
Haha, had exactly the same thoughts! I called it `run` before, but that one is already taken. We can overwrite it though

Reply:
I think we can replace run with this. I doubt anyone will notice and I dont think two need two ways of starting the server from the makefile.

Reply:
Yes! Done :)